### ISSUE TITLE
[Issue]:  Jellyfin not listen port, no WebGUI access.

### ISSUE THREAD

**Issue by pcela** [jellyfin/jellyfin#10179]
### Please describe your bug

Jellyfin stopped listening on web interface after Ubuntu update. A clean reinstall does nothing. I tested both stable and unstable version.

 netstat -naop | grep :8096
EMPTY

 netstat -natp | grep :8096
EMPTY

 netstat -naop | grep jelly
unix  2      [ ACC ]     STREAM     LISTENING     68364    21036/jellyfin       /tmp/dotnet-diagnostic-21036-457206-socket
unix  3      [ ]         STREAM     CONNECTED     68357    21036/jellyfin

 ps aux | grep jelly
jellyfin   21036  295  3.9 275023276 323384 ?    Ssl  13:51  24:50 /usr/bin/jellyfin --webdir=/usr/share/jellyfin/web --ffmpeg=/usr/lib/jellyfin-ffmpeg/ffmpeg

### Jellyfin Version

10.8.z

### if other:

 10.9.0.0

### Environment

```markdown
- OS: Ubuntu 23.04
- Linux Kernel: 6.2.0-31-generic
- Virtualization: none
- Clients: browser, Android, Kodi
- Browser: N/A
- FFmpeg Version: 5.1.3-Jellyfin
- Playback Method: N/A
- Hardware Acceleration: none
- GPU Model: none
- Plugins:
- Reverse Proxy: none, nginx
- Base URL: none
- Networking: host
- Storage: local/cloud
```

### Jellyfin logs

```shell
[2023-08-31 13:52:00.941 +02:00] [INF] Jellyfin version: "10.9.0"
[2023-08-31 13:52:01.070 +02:00] [INF] Environment Variables: ["[JELLYFIN_CONFIG_DIR, /etc/jellyfin]", "[JELLYFIN_WEB_OPT, --webdir=/usr/share/jellyfin/we
[2023-08-31 13:52:01.077 +02:00] [INF] Arguments: ["/usr/lib/jellyfin/bin/jellyfin.dll", "--webdir=/usr/share/jellyfin/web", "--ffmpeg=/usr/lib/jellyfin-f
[2023-08-31 13:52:01.078 +02:00] [INF] Operating system: "Linux 6.2.0-31-generic #31-Ubuntu SMP PREEMPT_DYNAMIC Mon Aug 14 13:42:26 UTC 2023"
[2023-08-31 13:52:01.078 +02:00] [INF] Architecture: X64
[2023-08-31 13:52:01.086 +02:00] [INF] 64-Bit Process: True
[2023-08-31 13:52:01.086 +02:00] [INF] User Interactive: True
[2023-08-31 13:52:01.086 +02:00] [INF] Processor count: 8
[2023-08-31 13:52:01.087 +02:00] [INF] Program data path: "/var/lib/jellyfin"
[2023-08-31 13:52:01.087 +02:00] [INF] Web resources path: "/usr/share/jellyfin/web"
[2023-08-31 13:52:01.088 +02:00] [INF] Application directory: "/usr/lib/jellyfin/bin/"
[2023-08-31 13:52:01.498 +02:00] [INF] Setting cache path: "/var/cache/jellyfin"
[2023-08-31 13:52:01.586 +02:00] [INF] Loading assemblies
[2023-08-31 13:52:01.851 +02:00] [INF] Defined LAN addresses: ["127.0.0.1/8", "10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"]
[2023-08-31 13:52:01.851 +02:00] [INF] Defined LAN exclusions: []
[2023-08-31 13:52:01.853 +02:00] [INF] Using LAN addresses: ["127.0.0.1/8", "10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"]
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hrvstr** [comment]
Any update on this issue? I am running Debian 12.1 (bookworm) and Jellyfin 10.8.11 on a Raspberry Pi 2  and can't get access to the web interface after a clean install of Debian and Jellyfin.

**Comment by srcrist** [comment]
Closing this issue as it does not reference a bug that we are able to reproduce in production or testing environments. The most likely cause would be misconfigured network settings. Please visit our support communities on the forum or our Matrix/Discord server if you'd like some help troubleshooting the issue.

**Comment by Bond-009** [comment]
I'm guessing this is the same issue as #10070 and will be fixed by #10051

**Comment by hrvstr** [comment]
Sorry for the late reply. I installed Jellyfin through `dietpi-software` from [dietpi.com](https://dietpi.com) which changes the default port to `8097` (due to conflicts with other software). Previously I had Jellyfin installed directly before it was available in `dietpi-software`. That's why I got confused because my bookmarks to the Web GUI stopped working after a reinstall.

See [https://dietpi.com/docs/software/media/#jellyfin](https://dietpi.com/docs/software/media/#jellyfin).
